cmd/dp-observation-importer/main.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"github.com/ONSdigital/dp-observation-importer/config"
+	"github.com/ONSdigital/dp-observation-importer/dimension"
+	"github.com/ONSdigital/dp-observation-importer/errors"
+	"github.com/ONSdigital/dp-observation-importer/event"
+	"github.com/ONSdigital/dp-observation-importer/observation"
+	"github.com/ONSdigital/go-ns/kafka"
+	"github.com/ONSdigital/go-ns/log"
+	bolt "github.com/johnnadratowski/golang-neo4j-bolt-driver"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+)
+
+func main() {
+	log.Namespace = "dp-observation-importer"
+
+	config, err := config.Get()
+	if err != nil {
+		log.Error(err, nil)
+		os.Exit(1)
+	}
+
+	// Avoid logging the neo4j URL as it may contain a password
+	log.Debug("loaded config", log.Data{
```

Comment:
A good suggestion - I just tried it but gofigure panic'd "reflect.Value.SetString using value obtained using unexported field" - will have to discuss this.